homeworks/src/homework17/CategoryException.java

```diff
@@ -0,0 +1,14 @@
+package homework17;
+
+public class CategoryException extends Throwable {
+
+    private String message;
+    public CategoryException (String category){
+        this.message = "The Product from category " + category + " is not found\n";
+    }
+
+    @Override
+    public String getMessage (){
```

Comment:
In 99% of real cases, you will extend `RuntimeException`
Hardly ever you will extend from `Exception`
And never from `Throuble`

Exceptions already includes message